azerothcore/azerothcore-wotlk
# fix(Scripts/Underbog): Moved Gha'zan script to C++ file.

Added scripted waypoints.
Fixes #13793



## Issues Addressed:

- Closes #13793

## Tests Performed:

- Tested ingame.

## How to Test the Changes:

`.go xyz 251.88 -434.55 81.372864 546 4.6`
Note where Ghaz'an spawns, see if it is swimming in a circle
go to areatrigger 4302 (`.go xyz 241.09 -373.76 72.447418 546 3.75`)
See if it moves towards the platform
When it reaches the platform, see if it wanders.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
